src/base/bittorrent/session.cpp

```diff
@@ -3149,6 +3149,17 @@ void Session::setAsyncIOThreads(const int num)
     configureDeferred();
 }
 
+int Session::stopTrackerTimeout() const
+{
+    return m_stopTrackerTimeout;
+}
+
+void Session::setStopTrackerTimeout(const int value)
+{
+    if (value != stopTrackerTimeout())
+        m_stopTrackerTimeout = value;
```

Comment:
missing `configureDeferred();` here.
You should copy the pattern in `Session::setAsyncIOThreads`.